### PR TITLE
Add words_to_watch and rebuild the model

### PR DESCRIPTION
**Pull request by mariushoch**
https://phabricator.wikimedia.org/T214091

### CODE REVIEW

File: model_info/enwiki.nettrom_wp10.md [hunk 1]
```diff
@@ -1,85 +1,85 @@
...
 	 - type: GradientBoosting
 	 - version: 0.8.1
-	 - params: {'random_state': None, 'learning_rate': 0.01, 'max_features': 'log2', 'scale': True, 'presort': 'auto', 'verbose': 0, 'criterion': 'friedman_mse', 'center': True, 'min_impurity_decrease': 0.0, 'multilabel': False, 'min_impurity_split': None, 'warm_start': False, 'subsample': 1.0, 'init': None, 'population_rates': None, 'max_depth': 3, 'max_leaf_nodes': None, 'loss': 'deviance', 'min_weight_fraction_leaf': 0.0, 'label_weights': None, 'n_estimators': 100, 'labels': ['Stub', 'Start', 'C', 'B', 'GA', 'FA'], 'min_samples_split': 2, 'min_samples_leaf': 1}
+	 - params: {'loss': 'deviance', 'presort': 'auto', 'labels': ['Stub', 'Start', 'C', 'B', 'GA', 'FA'], 'scale': True, 'min_samples_leaf': 1, 'min_impurity_split': None, 'criterion': 'friedman_mse', 'label_weights': None, 'learning_rate': 0.01, 'n_estimators': 100, 'warm_start': False, 'max_depth': 3, 'subsample': 1.0, 'center': True, 'min_impurity_decrease': 0.0, 'min_weight_fraction_leaf': 0.0, 'max_leaf_nodes': None, 'init': None, 'min_samples_split': 2, 'max_features': 'log2', 'population_rates': None, 'random_state': None, 'verbose': 0, 'multilabel': False}
```
Comment (halfak) [inline model_info/enwiki.nettrom_wp10.md]: It looks like the best GB model in tuning ran with `learning_rate=0.01, n_estimators=500, max_depth=7, max_features="log2"`  It doesn't look like those are the params being reported here.  Maybe the model would work better if we use these?

Comment (halfak) [reply model_info/enwiki.nettrom_wp10.md]: See https://github.com/wikimedia/articlequality/blob/master/Makefile#L105

Comment (mariushoch) [reply model_info/enwiki.nettrom_wp10.md]: As per the call, I changed this to the best performing parameters with `n_estimators = 100`.

**Comment by mariushoch** [comment]
With `n_estimators=100`:
`-rw-rw-r-- 1 hoo wikidev 9.1M Feb  5 03:19 enwiki.nettrom_wp10.gradient_boosting.model`

With `n_estimators=500`:
`-rw-rw-r-- 1 hoo wikidev  42M Feb  5 23:31 enwiki.nettrom_wp10.gradient_boosting.model`

**Comment by mariushoch** [comment]
I uploaded the `n_estimators=500` version now… if the model is to large, I'm fine with going back to a lower n_estimators number (there's also a fairly good GradientBoosting run in the tuning report with `learning_rate=0.01, n_estimators=300, max_depth=7, max_features="log2"`).

**Comment by halfak** [comment]
Thank you.  This is a hard call.  That file size difference is substantial, but not overwhelming.  We have other models that are over 100MB.  The fitness gains may be worthwhile, but it's hard to tell without digging through a lot of predictions.  That said, this won't make ORES fall over and the I think it's reasonable to seek a bit more performance at the cost of RAM.  Let's stick with `n_estimators=500`.  Thanks for your work on this.

**Pull request merged**